tikv-client/src/main/java/com/pingcap/tikv/codec/CodecDataInput.java

```diff
@@ -16,7 +16,11 @@
 package com.pingcap.tikv.codec;
 
 import com.google.protobuf.ByteString;
-import java.io.*;
+import java.io.ByteArrayInputStream;
+import java.io.DataInput;
+import java.io.DataInputStream;
+import java.io.IOException;
+import java.io.InputStream;
```

Comment:
any reason?

Reply:
Do you mean any reason for importing them?

Reply:
change from `*` ?

Reply:
I really don't know..

but it seems that this way looks better?